recipes/cytoolz/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "0.7.5" %}
+
+package:
+  name: cytoolz
+  version: {{ version }}
+
+source:
+  fn: cytoolz-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/c/cytoolz/cytoolz-{{ version }}.tar.gz
+  md5: 15fbc1c56fca80cbca0e89f37c93d432
+
+build:
+  number: 0
+  script: python setup.py install --record=record.txt
+
+requirements:
+  build:
+    - setuptools
+    - python
+    - cython
+    - toolz
+    - libpython  # [win and not (win64 and py34)]
```

Comment:
This was just copied over from the existing recipe at https://github.com/pytoolz/cytoolz/blob/master/conda.recipe/meta.yaml#L14. Cytoolz uses some internal libpython functions not exposed by cython directly, so I assume that's the reason. @eriknw probably knows more.